tests/Command/ContentHubHealthCheckWebhookStatusTest.php

```diff
@@ -26,6 +27,8 @@
  */
 class ContentHubHealthCheckWebhookStatusTest extends ContentHubCommandTestBase {
 
+  use ProphecyTrait;
+
```

Comment:
Better to add this in ContentHubCommandTestBase than adding in all the inheritors.